src/app/application/components/app-collection/AppCollection.vue

```diff
@@ -71,6 +76,13 @@ const slots = useSlots()
 
 const items = ref<unknown[] | undefined>(props.items)
 const cacheKey = ref<number>(0)
+
+const remainingSlots = computed(() => {
+  const { toolbar, ...remainingSlots } = slots
+
+  return remainingSlots
+})
+
```

Comment:
If we can either do the `key === 'toolbar'`, or even better just reuse the `.k-table-toolbar` selector, when we can get rid of this code.

Reply:
See https://github.com/kumahq/kuma-gui/pull/1203#discussion_r1267649374 for why I’m not using the existing `.k-table-toolbar` selector for these styles.

Reply:
Just to note for history that we are continuing this conversation in the single place that @kleinfreund referred to.